streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -938,6 +938,9 @@
         // Private API used to disable the fix on left/outer joins (https://issues.apache.org/jira/browse/KAFKA-10847)
         public static final String ENABLE_KSTREAMS_OUTER_JOIN_SPURIOUS_RESULTS_FIX = "__enable.kstreams.outer.join.spurious.results.fix__";
 
+        // Private API used to control the emit latency for left/outer join results (https://issues.apache.org/jira/browse/KAFKA-10847)
+        public static final String EMIT_INTERVAL_MS_KSTREAMS_OUTER_JOIN_SPURIOUS_RESULTS_FIX = "emit.interval.ms.kstreams.outer.join.spurious.results.fix__";
```

Comment:
Should we add underscore in front of the config like the above ones? i.e. `__emit.interval.ms.kstreams.outer.join.spurious.results.fix__`

Reply:
Good catch!